README.adoc

```diff
@@ -197,6 +197,17 @@ serviceAccounts:
 
 For each ServiceAccount defined in this section, the `ksctl generate cli-configs` generates a separate `ksctl.yaml` file with the corresponding cluster configuration and tokens. As an administrator of the clusters, run this command and distribute securely the generated `ksctl.yaml` files to other team members.
 
+====== Testing the `ksctl generate cli-configs` command locally
+1. Run `make install`
+2. Create `kubesaw-admins.yaml` (as an example, check link:test-resources/dummy.openshiftapps.com/kubesaw-admins.yaml[kubesaw-admins.yaml])
+3. Run `ksctl generate admin-manifests --kubesaw-admins <path>/kubesaw-admins.yaml --out-dir <admin-manifests-out-dir-path>`
+4. Create resources from the `<admin-manifests-out-dir-path>` of the previous command. Please, note that you will need to create some namespaces manually (`oc create ns <namespace-name>`), such as `sandbox-sre-host`, `first-component`, `second-component`, `some-component`, `sandbox-sre-member`, and `crw`, for example.
```

Comment:
The missing namespaces is not ideal. We could maybe add a flag to `adm generate admin-manifests` command so it includes the namespace manifests as well. 
But that's for later, for now it's good to have it documented

Reply:
Deal, thanks :)